### DISCUSSION TITLE
Total Newb, pls help

### DISCUSSION THREAD

**Discussion by NyoX01** [bassamanator/Sovol-SV06-firmware#63] [General]
I've gotten as far as method 1.  Klipper is installed on the printer and I can pull it up on my pc.  I've set the printer port in the printer.cfg. Just not sure how to go about doing method 1 or two for that matter.  I have the file downloaded on my pc.

**Comment by bassamanator** [comment]
I recommend Method 1. You need to ssh into the Rpi. If on mac/linux, very easy `ssh pi@address`. On windows you need to use putty.

**Reply by NyoX01** [reply]
I'm in there. So I need to take the contents of printer.cfg and move them to printer.cfg in klipper?

**Reply by NyoX01** [reply]
or rm the printer_data/config?

**Reply by NyoX01** [reply]
I guess I'm not sure if I use that command, it will erase the config completely. Or maybe that's what we want then do #3 on method 1...

**Reply by bassamanator** [reply]
https://talky.io/against-a-red-frog

**Reply by NyoX01** [reply]
Hey my good friend Bassamanator, could we perhaps chat again?

**Comment by bassamanator** [comment]
Klipper has already been installed. You can resume where we left off yesterday, Initial Steps > Step 2. Just follow instructions from Step 2 on wards.

**Reply by NyoX01** [reply]
No, my sd card shit the bed on me this morning lol. So I had to go through everything all over again.  But I did it!!!! My SV06 Plus is fully Klipperized!! Thank you so much! Bassam for President! You have my vote friend. Thanks so much!

**Reply by bassamanator** [reply]
Easy peasy, lemon squeezey. 

Welcome to the world of Klipper, and more importantly, linux! One of my goals with all this was to get people out of their comfort zone, I think I've succeeded. All the best Justin!
